recipes/hisat2-pipeline/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "hisat2-pipeline" %}
+{% set version = "1.0.6" %}
+{% set sha256 = "cc218d21ba1443a6581a2934afef2434d26aaac4bdc974b9747341fd1bac9fd7" %}
+{% set DOI = "https://doi.org/10.5281/zenodo.10389729" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+  run_exports:
+  - {{ pin_subpackage('hisat2-pipeline', max_pin="x.x") }}
+  
+
+source:
+  url: https://github.com/mcamagna/HISAT2-pipeline/archive/refs/tags/v{{ version }}-bioconda.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  host:
+  - python >=3
+  - pip
+  - hisat2
+  - stringtie
+  - samtools >=1.15
+  - pandas
+  - openpyxl
+  run:
+  - python >=3
+  - pandas
+
+test:
+  commands:
+  - hisat2-pipeline --version
+  imports:
+  - pandas
+  
+about:
+  home: https://github.com/mcamagna/HISAT2-pipeline
+  license: GPLv3
```

Comment:
please use SPDX identifier https://spdx.org/licenses/GPL-3.0-or-later.html

Reply:
Thank you for the review. I've replaced the license string with the approriate SPDX identifier.